docs/winston.asciidoc

```diff
@@ -265,3 +265,5 @@ const logger = winston.createLogger({
   // ...
 })
 ----
+
+To learn more, try out our tutorial using Node.js ECS logging with winston: {cloud}/ec-getting-started-search-use-cases-node-logs.html[Ingest logs from a Node.js web application using Filebeat].
```

Comment:
I think this would be better in the "Setup" section up higher. Could you move this up to just above these lines:

```
[float]
[[winston-usage]]
=== Usage
```

?

Reply:
Good idea. I moved the link up to the "Configure Filebeat" section.